src/main/java/org/mockito/plugins/DoNotMockEnforcerWithType.java

```diff
@@ -0,0 +1,20 @@
+package org.mockito.plugins;
```

Comment:
The new files are missing license headers (see checkstyle failure on CI). Please add the license header

```
/*
 * Copyright (c) 2024 Mockito contributors
 * This program is made available under the terms of the MIT License.
 */
```

Reply:
@TimvdLippe Done